tests/integration/ha/test_backups.py

```diff
@@ -365,49 +345,6 @@ async def test_remove_and_readd_s3_relation(ops_test: OpsTest) -> None:
         idle_period=IDLE_PERIOD,
     )
 
-    leader_id = await get_leader_unit_id(ops_test, app)
-
-    action = await run_action(ops_test, leader_id, "create-backup")
-    logger.info(f"create-backup output: {action}")
-
-    assert action.status == "completed"
-
-    list_backups = await run_action(ops_test, leader_id, "list-backups", params={"output": "json"})
-    logger.info(f"list-backups output: {list_backups}")
-
-    # Expected format:
-    # namespace(status='completed', response={'return-code': 0, 'backups': '{"1": ...}'})
-    backups = json.loads(list_backups.response["backups"])
-    assert list_backups.status == "completed"
-    assert len(backups.keys()) == int(action.response["backup-id"])
-    assert backups[action.response["backup-id"]]["state"] == "SUCCESS"
-
-    await http_request(
-        ops_test,
-        "DELETE",
-        f"https://{leader_unit_ip}:9200/{TEST_BACKUP_INDEX}",
-        app=app,
-    )
-
-    action = await run_action(
-        ops_test, leader_id, "restore", params={"backup-id": int(action.response["backup-id"])}
-    )
-    logger.info(f"restore-backup output: {action}")
-    assert action.status == "completed"
-
-    # index document
-    doc_id = TEST_BACKUP_DOC_ID
-    # check that the doc can be retrieved from any node
-    logger.info("Test backup index: searching")
-    for u_id, u_ip in units.items():
-        docs = await search(
-            ops_test,
-            app,
-            u_ip,
-            TEST_BACKUP_INDEX,
-            query={"query": {"term": {"_id": doc_id}}},
-            preference="_only_local",
-        )
-        # Validate the index and document are present
-        assert len(docs) == 1
-        assert docs[0]["_source"] == default_doc(TEST_BACKUP_INDEX, doc_id)
```

Comment:
Here again I find this check that I don't see anywhere migrated...

Reply:
@juditnovak it is not gone, but this check is done in the _backup and _restore tests already.